bench/basic-app/app/api/app/route.js

```diff
@@ -1,5 +1,5 @@
 export function GET() {
-  return { name: 'John Doe' }
+  return new Response(JSON.stringify({ name: 'John Doe' }))
```

Comment:
```suggestion
  return Response.json(({ name: 'John Doe' })
```
We probably want to use this for correct content-type header as well